sdk/monitor/monitor-opentelemetry-exporter/src/export/statsbeat/types.ts

```diff
@@ -53,6 +57,38 @@ export class NetworkStatsbeat {
   }
 }
 
+/**
+ * Statsbeat class for customer-visible telemetry.
+ * @internal
+ */
+export class CustomerStatsbeat {
+  public endpoint: string;
+
+  public host: string;
+
+  public totalItemSuccessCount: { count: number; telemetry_type: TelemetryType }[];
```

Comment:
Why is this an array and not just a mapping of { `telemetry_type: count` }?

Reply:
Each CustomerStatsbeat class can have multiple count -> telemetry_type mappings, so an example structure could be:
[
  { count: 5, telemetry_type: DEPENDENCY },
  { count: 10, telemetry_type: REQUEST }
]